arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -1516,6 +1709,37 @@ Result IResearchDataStore::remove(transaction::Methods& trx,
 
   return {};
 }
+
+bool IResearchDataStore::exists(IResearchDataStore::Snapshot const& snapshot,
+                                LocalDocumentId documentId, bool nested,
+                                TRI_voc_tick_t const* recoveryTick) const {
+  if (recoveryTick == nullptr) {
+    TRI_ASSERT(false) << "Call exists not in recovery";
```

Comment:
Why it is wrong to call this method not in the recovery?

Reply:
Probably nothing, but now we don't have such code

Reply:
Then maybe remove this assert?  Or at least  please leave a comment, why this assert is here

Reply:
Ok